src/test/java/org/opensearch/neuralsearch/e2e/HybridCollapseIT.java

```diff
@@ -33,12 +33,27 @@ public class HybridCollapseIT extends BaseNeuralSearchIT {
     @Before
     public void setUp() throws Exception {
         super.setUp();
-        createTestIndex();
-        indexTestDocuments();
         createSearchPipeline(SEARCH_PIPELINE, "min_max", "arithmetic_mean", Map.of());
     }
 
-    public void testCollapse_whenE2E_thenSuccessful() {
+    public void testCollapse_withSingleShard_thenSuccessful() {
+        createTestIndex(1);
```

Comment:
can you use constant and name it in a way it's clear we are passing number of shards, like NUMBER_OF_SHARDS_ONE and NUMBER_OF_SHARDS_FIVE. Alternatively you can create a wrapper method that will set required number of shards, like
```
createTestIndexWithFiveShards() {
   createTestIndex(5);
}
```